data/Kabit Savaiye Bhai Gurdas Ji/235.json

```
@@ -14,7 +14,7 @@
         "pronunciation_information": null,
         "type": "Pankti",
         "gurmukhi": {
-          "Seva Singh": "jYsy mnu DwvY pr qn Dn dUKnw lau; sRI gur srin swDsMg lau n AwveI ["
+          "Seva Singh": "jYso mnu DwvY pr-qn Dn dUKn lau, sRI gur srin swDsMg lau n AwveI ]"
```

```suggestion
          "Seva Singh": "jYso mnu DwvY pr-qn Dn dUKn lau; sRI gur srin swD sMig lau n AwveI ]"
```

The comma in the source is used as the main (heavy) pause. Our `;` semi-colon symbol represents the hard/heavy/main pause/vishram.

In addition, adding the space and sihari for `sadh sang`.